.github/workflows/ci.yml

```diff
@@ -737,3 +737,87 @@ jobs:
         ./util/opensslwrap.sh version -c
     - name: test external pyca
       run: make test TESTS="test_external_pyca" VERBOSE=1
+
+  openbsd-x86_64:
+    runs-on: ubuntu-latest
+    name: Test on OpenBSD (amd64)
+    steps:
+    - uses: actions/checkout@v4
+    - name: config
+      uses: cross-platform-actions/action@fe0167d8082ac584754ef3ffb567fded22642c7d #v0.27.0
+      with:
+        operating_system: openbsd
+        architecture: x86-64
+        version: '7.7'
```

Comment:
Support for OpenBSD 7.7 was added in version 0.28.0 of the GitHub action [1]. Latest version is 0.29.0.

[1] https://github.com/cross-platform-actions/action/blob/master/changelog.md#0280---2025-05-19